src/test/java/com/hillel/items_exchange/controller/UserControllerTest.java

```diff
@@ -133,4 +134,220 @@ private ResultActions getResultActions(HttpMethod httpMethod, String path,
+                .andExpect(status().isOk());
+    }
+
+    @Test
+    @WithMockUser(username = "admin")
+    @Transactional
+    @DataSet("database_init.yml")
+    @ExpectedDataSet(value = "children/update.yml")
+    void updateChild_Success() throws Exception {
+        mockMvc.perform(put("/user/child/")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(ChildDtoCreatingUtil.getValidChildDtoForUpdate())
+                .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isOk());
+    }
+
+    @Test
+    @WithMockUser(username = "admin")
+    @Transactional
+    @DataSet("database_init.yml")
+    void addChild_Fail_EmptyDto() throws Exception {
+        mockMvc.perform(post("/user/child/")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content("")
+                .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isBadRequest());
+    }
+
+    @Test
+    @WithMockUser(username = "admin")
+    @Transactional
+    @DataSet("database_init.yml")
+    void removeChild_Fail_EmptyDto() throws Exception {
+        mockMvc.perform(delete("/user/child/")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content("")
+                .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isBadRequest());
+    }
+
+    @Test
+    @WithMockUser(username = "admin")
+    @Transactional
+    @DataSet("database_init.yml")
+    void updateChild_Fail_EmptyDto() throws Exception {
+        mockMvc.perform(put("/user/child/")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content("")
+                .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isBadRequest());
+    }
+
+    @Test
+    @WithMockUser(username = "no-such-user")
+    @Transactional
+    @DataSet("database_init.yml")
+    void addChild_Fail_AbsentUser() throws Exception {
+        mockMvc.perform(post("/user/child/")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(ChildDtoCreatingUtil.getValidChildDtoForCreate())
+                .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isNotFound());
+    }
+
+    @Test
+    @WithMockUser(username = "no-such-user")
+    @Transactional
+    @DataSet("database_init.yml")
+    void removeChild_Fail_AbsentUser() throws Exception {
+        mockMvc.perform(delete("/user/child/")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(ChildDtoCreatingUtil.getValidChildDtoForCreate())
+                .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isBadRequest());
+    }
+
+    @Test
+    @WithMockUser(username = "no-such-user")
```

Comment:
Keep such kind of tests only if you created the code that would validate such things.

Reply:
But I already validate this. I am checking is such user present and throwing UsernameNotFoundException if he does not. Is this is what you talking about?